### PR TITLE
Migrate to pytest

### PR DESCRIPTION
**Pull request by kurayami07734**
### Brief Summary of Changes
edited the test command in tox.ini and include pytest in setup.py

#### What does this PR address?
- [ ] GitHub issue ( #368  )
- [x] Refactoring
- [ ] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

**Comment by kurayami07734** [comment]
How do I include pytest as a dependency?

File: tox.ini [hunk 1]
```diff
@@ -6,7 +6,7 @@ envlist =
 [testenv]
 usedevelop = True
 commands =
-  core: python setup.py test {env:P_ARGS:}
+  core: python setup.py && python -m pytest test
```
Comment (const-cloudinary) [inline tox.ini]: According to:
https://docs.pytest.org/en/7.1.x/how-to/usage.html#calling-pytest-through-python-m-pytest

the command should be:
`python -m pytest test` 
without `python setup.py &&` part

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (const-cloudinary) [inline tox.ini:9]:
```suggestion
  core: python -m pytest test 
```

**Review by const-cloudinary — APPROVED** [review]

**Comment by const-cloudinary** [comment]
Fixed in #404

**Pull request closed without merging**